File: TaxiService/Taxi.cs
using System;

public class Taxi
{
    public int Id { get; set; }
    public int SeatCapacity { get; set; }
    public MyCalendar Calendar { get; set; }

    public Taxi(int id, int seatCapacity)
    {
        Id = id;
        SeatCapacity = seatCapacity;
        Calendar = new MyCalendar();
    }
}

File: Taxi.java
public class Taxi {
    int id;
    int seatCapacity;
    MyCalendar calendar;

    public Taxi(int id,int seatCapacity) {
        this.id = id;
        this.seatCapacity = seatCapacity;
        this.calendar = new MyCalendar();
    }
}
